sdk/translation/Azure.AI.Translation.Text/src/Custom/TextTranslationTranslateOptions.cs

```diff
@@ -0,0 +1,175 @@
+        /// project details to this parameter to use your deployed customized system. Default value is: general.
+        /// </summary>
+        public string Category { get; set; }
+        /// <summary>
+        /// Specifies how profanities should be treated in translations.
+        /// Possible values are: NoAction (default), Marked or Deleted.
+        /// </summary>
+        public ProfanityAction? ProfanityAction { get; set; }
+        /// <summary>
+        /// Specifies how profanities should be marked in translations.
+        /// Possible values are: Asterisk (default) or Tag.
+        /// </summary>
+        public ProfanityMarker? ProfanityMarker { get; set; }
+        /// <summary>
+        /// Specifies whether to include alignment projection from source text to translated text.
+        /// Possible values are: true or false (default).
+        /// </summary>
+        public bool? IncludeAlignment { get; set; }
+        /// <summary>
+        /// Specifies whether to include sentence boundaries for the input text and the translated text.
+        /// Possible values are: true or false (default).
+        /// </summary>
+        public bool? IncludeSentenceLength { get; set; }
+        /// <summary>
+        /// Specifies a fallback language if the language of the input text can't be identified.
+        /// Language autodetection is applied when the from parameter is omitted. If detection fails,
+        /// the SuggestedFrom language will be assumed.
+        /// </summary>
+        public string SuggestedFrom { get; set; }
+        /// <summary>
+        /// Specifies the script of the input text.
+        /// </summary>
+        public string FromScript { get; set; }
+        /// <summary>
+        /// Specifies the script of the translated text.
+        /// </summary>
+        public string ToScript { get; set; }
+        /// <summary>
+        /// Specifies that the service is allowed to fall back to a general system when a custom system doesn&apos;t exist.
+        /// Possible values are: true (default) or false.
+        ///
+        /// AllowFallback=false specifies that the translation should only use systems trained for the category specified
+        /// by the request. If a translation for language X to language Y requires chaining through a pivot language E,
+        /// then all the systems in the chain (X → E and E → Y) will need to be custom and have the same category.
+        /// If no system is found with the specific category, the request will return a 400 status code. AllowFallback=true
+        /// specifies that the service is allowed to fall back to a general system when a custom system doesn't exist.
+        /// </summary>
+        public bool? AllowFallback { get; set; }
+
+        /// <summary> Initializes new instance of TextTranslationTranslateOptions. </summary>
+        public TextTranslationTranslateOptions(): base()
```

Comment:
Same comment as I provided in Transliterations Options.

Reply:
The error is in the use of a base class, that was not required.  I have corrected that.  The constructors with no parameters are consistent with the Document Translation and other services.  In practice if they are created with no properties they will fail parameter checks in the Translate/Transliterate methods.  All of the samples and now the tests as well use the constructor with property values like this:

TextTranslationTranslateOptions options = new TextTranslationTranslateOptions()
{
    FromScript = "Latn",
    SourceLanguage = "ar",
    ToScript = "Latn",
    TargetLanguages = new[] { "zh-Hans" },
    Content = new[]
    {
        "hudha akhtabar."
    }
};

Reply:
Removed in next commit.